src/puppetlabs/trapperkeeper/authorization/ring_middleware.clj

```diff
@@ -209,11 +209,14 @@
    allow-header-cert-info :- schema/Bool
    oid-map :- acl/OIDMap]
   (if-let [cert (request->cert request allow-header-cert-info)]
-    (let [extensions (ssl-utils/get-extensions cert)
+    (let [oid-map (merge acl/default-oid-map oid-map)
```

Comment:
I thought of that... My concern was that the functions being called (like allowed) can totally be called in isolation outside of the middleware and as such I wanted the defaults applied at the lowest levels, not the highest level. I found the two lowest-level uses of the oid-map and did the merging there to make sure there wasn't a possible call site that would miss the defaults.

Does that make sense / seem reasonable? Obviously the downside is if another low-level use crops up, someone has to know to do this, so I'm not 100% convinced I'm right here.